### PR TITLE
Routes & DB support for Monsters and Attacks

### PR DESCRIPTION
**Pull request by villejacob**

### CODE REVIEW

File: actions/user_data.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	}
+
+	return c.Render(200, render.JSON(map[string]interface{}{
+		"id": user.ID,
+		"email": user.Email,
+		"monsters": user.Monsters,
+	}))
+}
+
+func userAnimationsHandler(c buffalo.Context) error {
```
Comment (Joepena) [inline actions/user_data.go]: This procedure is awesome. 👍

File: grifts/dex.json [hunk 2]
```diff
@@ -0,0 +1,164 @@
+{
```
Comment (Joepena) [inline grifts/dex.json]: Good seed file I will make a PR to seed this soon

File: models/attack.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package models
+
+import "errors"
```
Comment (Joepena) [inline models/attack.go]: `github.com/pkg/errors` is what we have as an import in other files. Maybe we should use one of them uniformly? thoughts?

File: models/monster.go [hunk 4]
```diff
@@ -0,0 +1,24 @@
+package models
+
+import "errors"
```
Comment (Joepena) [inline models/monster.go]: same as previous comment: `github.com/pkg/errors` vs `errors`

File: actions/user_data.go (hunk 1 above)
Comment (villejacob) [reply actions/user_data.go]: 😘

**Pull request merged**